prow/pjutil/pjutil.go

```diff
@@ -278,3 +279,11 @@ func JobURL(plank config.Plank, pj prowapi.ProwJob, log *logrus.Entry) string {
 	}
 	return ""
 }
+
+// ClusterToCtx converts the prow job's cluster to a cluster context
+func ClusterToCtx(cluster string) string {
+	if cluster == kube.InClusterContext {
+		return kube.DefaultClusterAlias
```

Comment:
Do we validate that two aliases not point to the same place when someone provides explicit config?

Reply:
Not right now